config/config.go

```diff
@@ -70,8 +72,8 @@ type Config struct {
 	// Config customized Docker Sock path
 	DockerSockPath string `json:"dockerSockPath,omitempty"`
 
-	// Default workflow spec, will be adde to workflow if the parameters are not set in the workflow
-	DefautWorkflowSpec *wfv1.WorkflowSpec `json:"workflowDefaults,omitempty"`
+	// WorkflowDefaults are values that will apply to all Workflows from this controller, unless overridden on the Workflow-level
+	WorkflowDefaults *wfv1.WorkflowSpec `json:"workflowDefaults,omitempty"`
```

Comment:
I think this should either stay as `workflowDefaultsSpec`, or be generalized to `workflowDefaults` and also copy labels and annotations.

Reply:
Great point. I think generalizing would be more powerful, I'll work on this.